.github/workflows/zcash-params.yml

```diff
@@ -14,98 +14,18 @@ on:
       - 'docker/zcash-params/Dockerfile'
       - '.github/workflows/zcash-params.yml'
```

Comment:
Blocker: please add all the dependent workflow files to all instances of the build filter paths.